build.assets/tooling/cmd/trigger-workflow/args.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2022 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/gravitational/trace"
+)
+
+// inputMap represents thi inoput values to a workflow. The odd `interface{}`
+// value type (rather than the mor eobvious `string`) is so that we match the
+// type that the GitHub API expects, and we can plug an `inputMap` straight
+// in without any further conversion.
+type inputMap map[string]interface{}
+
+func (m inputMap) String() string {
+	text := strings.Builder{}
+	for k, v := range m {
+		fmt.Fprintf(&text, "%s=%v, ", k, v)
+	}
+	return text.String()
+}
+
+func (m inputMap) Set(s string) error {
+	parts := strings.SplitN(s, "=", 2)
+
+	key := parts[0]
+	if key == "" {
+		return trace.BadParameter("missing input name")
+	}
+
+	value := ""
+	if len(parts) > 1 {
+		value = parts[1]
+	}
```

Comment:
nit: catch any bad input here in an `else` so it doesn't need to be debugged further down the line?